### DISCUSSION TITLE
👍

### DISCUSSION THREAD

**Discussion by Simsopheap** [primefaces/community#1460] [PrimeNG]
this.items = [
      {
        label: 'File',
        icon: 'pi pi-fw pi-file',
        items: [
          {
            label: 'New 1',
            icon: 'pi pi-fw pi-plus',
          },
          {
            label: 'New 2',
            icon: 'pi pi-fw pi-plus',
          },
          {
            label: 'New 3',
            icon: 'pi pi-fw pi-plus',
            items: [
              {
                label: 'Bookmark',
                icon: 'pi pi-fw pi-bookmark'
              },
              {
                label: 'Video',
                icon: 'pi pi-fw pi-video',
                items: [
                  {
                    label: 'Bookmark',
                    icon: 'pi pi-fw pi-bookmark'
                  },
                  {
                    label: 'Video',
                    icon: 'pi pi-fw pi-video',
                    items: [
                      {
                        label: 'Bookmark',
                        icon: 'pi pi-fw pi-bookmark'
                      },
                      {
                        label: 'Video',
                        icon: 'pi pi-fw pi-video',
                        items: [
                          {
                            label: 'Bookmark',
                            icon: 'pi pi-fw pi-bookmark'
                          },
                          {
                            label: 'Video',
                            icon: 'pi pi-fw pi-video'
                          }
                        ]
                      }
                    ]
                  }
                ]
              }
            ]
          },
        ]
      }
    ];

<p-panelMenu [model]="items" [style]="{'width':'300px'}"></p-panelMenu>

**Comment by cetincakiroglu** [comment]
Closing the discussion since it's invalid.
